### PR TITLE
Amplitude Connector

### LINKED ISSUE #1945: Amplitude

### PR DESCRIPTION
**Pull request by soundayra21**
Closes #1945 

### Code Changes
NEW SaaS Connector Amplitude for access and erasure 

### Steps to Confirm
* [x] Add/configure the Amplitude connector in UI
* [ ] Verify Amplitude config and icon UI looks good 
* [x] Add user data using Amplitude api
* [x] Create an Access DSR with the new Amplitude user email 
* [x] Verify email sent by fides with user data is correct 
* [x] Create an Delete DSR with the new Amplitude user email 
* [x] Verify the user data is now deleted in Amplitude

### Pre-Merge Checklist
* [ ] All CI Pipelines Succeeded
* [ ] Update `CHANGELOG.md`

### SaaS Connector Checklist 
* [x] [Spec doc link](https://ethyca.atlassian.net/wiki/spaces/EN/pages/2606006273/Amplitude+Connector+Specification)
* [x] Vault Creds
* [x] Postman Collection
* [x] SaaS Config And Dataset 
* [x] Access and Erasure Tests Passing 
* [x] Icon

### CODE REVIEW

**Comment by soundayra21** [comment]
image
image
image

**Comment by soundayra21** [comment]
Post to deletion/users takes days to complete

File: data/saas/config/amplitude_config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+        strategy: api_key
+        configuration:
+          headers:
+            - name: Authorization
+              value: Basic <authorization_key>
```
Comment (galvana) [inline data/saas/config/amplitude_config.yml:21]: The docs mention `api_key` and `secret_key`, let's update this to use those instead. Make sure to update the Postman collection and the `SaaS Connector Secrets` entry for Amplitude with these values.

**Review by galvana — CHANGES_REQUESTED** [review]
The values used for authentication need to be updated to match the docs. Also make sure to update Postman and 1password with these values.

File: data/saas/config/amplitude_config.yml (hunk 1 above)
Comment (soundayra21) [reply data/saas/config/amplitude_config.yml]: Changes the header authorisation to basic authentication with api_key as username and secret key as password. And also updated saas secrets and in 1password

**Review by Kelsey-Ethyca — APPROVED** [review]

**Pull request merged**